Judger/src/main/java/tw/waterball/judgegirl/judger/CCJudger.java

```diff
@@ -189,6 +189,10 @@ protected void onBeforeRunningTestcase(Testcase testcase) {
           and compare it after running the testcase.
           (The files-difference must be 'std.out', 'std.err' and out-files)
          */
+        Path path=getTestcaseOutputHome(testcase).getPath().resolve("std.out");
```

Comment:
Please follow the format.
` Path testcaseOutputHomePath = getTestcaseOutputHome(testcase).getPath().resolve("std.out");`